README.md

```diff
@@ -50,30 +50,32 @@ For Microverse based in these [requirements](https://www.theodinproject.com/cour
 
 # Features
 
-[![Framework][badge-framework]][framework-url]
-![javascript][]
+[![][javascript]][javascript-url] 
+[![][react]][react-url] 
```

Comment:
What about Next.js and Tailwind?

Reply:
yes, would be fine to add to the readme as a label